### DISCUSSION TITLE
如何快速将原本使用尾注的文章，转换为脚注（页下注）的形式？

### DISCUSSION THREAD

**Discussion by prong879** [zotero-chinese/wiki#287] [求助 / Q&A]
我已经撰写好了一篇文章，使用的是尾注的csl样式。最近准备给某期刊投稿，但是这个期刊要求使用页下注（脚注），于是我下载了使用脚注的csl，并且设置类型为脚注，然而就在文章从尾注向脚注转化之后，我文中从第二篇引用文献开始，全部从正文中消失，集中出现在最后一页。
![image](https://github.com/user-attachments/assets/8b3efe50-cb91-4fb1-939d-b8edd83f778c)
请问是否是我的操作有误？是否有较便捷的方法能快速把尾注转化为页下注（脚注）？

**Comment by zepinglee** [comment]
具体用了那两个样式？

可以尝试切换为别的样式，比如 numeric 或 author-date，刷新，再切换回去、刷新。
